README.md

```diff
@@ -52,7 +52,7 @@ What commands and how are they used to run the code.
 python main.py
 ```
 
-Mohith.M
-1RN17CS053
+Ankit Dwivedi
```

Comment:
You don't need to add your name under this repository. All your contributions are automatically managed by GitHub so you can go ahead and delete this.

Reply:
Alright